docs/buildah-run.md

```diff
@@ -26,7 +26,9 @@ The *path* to an alternate OCI-compatible runtime.
 
 Adds global flags for the container runtime. To list the supported flags, please
 consult manpages of your selected container runtime (`runc` is the default
-runtime, the manpage to consult is `runc(8)`)
+runtime, the manpage to consult is `runc(8)`).
+Note: Do not pass the leading `--` to the flag. If you want to pass `--log-format json`
```

Comment:
Nit of a nit.  I know I've added some, but I'd really rather not have the word 'you' in a man page.  (Note to self, scrub man pages).  Can I be a bear and ask that you change this to:
"To pass the runc flag `--log-format json` to `buildah run`,  the option given would be `--runtime-flag log-format=json`.

Reply:
My suggestion would replace the line starting with "If you want to pass", the preceding sentence is fine.

Reply:
fixed.